invenio_cli/helpers/commands.py

```diff
@@ -141,6 +144,108 @@ def build(self, pre, lock):
         self._symlink_templates()
         self.update_statics_and_assets(install=True)
 
+    def _containers_are_running(self):
+        """Ensures containers are running."""
+        click.secho('Making sure containers are up...', fg='green')
+        docker_helper = DockerHelper(local=True)
+        docker_helper.start_containers()
+        # TODO: Find faster way to procede when containers are ready
+        time.sleep(30)  # Give time to the containers to start properly
+
+    def services(self, force):
+        """Local start of containers (services).
+
+        NOTE: We use check=True to mimic set -e from original setup script
+              i.e. if a command fails, an exception is raised
+        """
+        self._containers_are_running()
```

Comment:
Minor: Probably a bit more descriptive name would be `ensure_containers_running` or `start_and_wait_containers`?

Reply:
`ensure_containers_running` :+1: